library-impl/src/main/scala/special/collection/CollsOverArrays.scala

```diff
@@ -305,7 +305,7 @@ class CollOverArrayBuilder extends CollBuilder {
       val rs = emptyColl(pt.tSnd)
       asColl[T](pairColl(ls, rs))
     case _ =>
-      new CollOverArray[T](Array[T]())
+      new CollOverArray[T](cT.emptyArray)
```

Comment:
ScalaDoc for the method (at least) missed

Reply:
This is override, which means the ScalaDoc is in the super class.